Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -219,6 +220,51 @@ public void TrustRootCertificate()
         }
 
         /// <summary>
+        /// Puts the certificate to the local machine's certificate store. 
+        /// Needs elevated permission. Works only on Windows.
+        /// </summary>
+        /// <returns></returns>
+        public bool TrustRootCertificateAsAdministrator()
+        {
+            if (RunTime.IsRunningOnMono())
+            {
+                return false;
+            }
+
+            var fileName = Path.GetTempFileName();
+            File.WriteAllBytes(fileName, RootCertificate.Export(X509ContentType.Pkcs12));
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "certutil.exe",
+                Arguments = "-importPFX -Enterprise -p \"\" -f \"" + fileName + "\"",
```

Comment:
In Linux an executable file can have any extension. So it will work with .exe extension, too. But I don't know how the certificates are working on Linux, maybe on Linux no elevated permission is needed for trusting the a root certificate for Firefox. I don't have Linux and my program which uses TWP is not working on Linux, so for me it is not a problem:)